### ISSUE TITLE
[DRAFT] Hidden Patterns on my life

### ISSUE THREAD

**Issue by josancamon19** [BasedHardware/omi#386]
**Is your feature request related to a problem? Please describe.**

**Show me Hidden Patterns in my Life**
I want my Limitless Pendant to recognize personal patterns over a period of time based on stuff I say, experience, etc. And then I want there to be a "Hub" where I can see these patterns and provide feedback (helpful/not helpful).

For example, maybe I talk way too much when I have 1:1s with co-workers. I want my Pendant to tell me that. Or maybe my mood tends to be worse when I wake up before 7am. Or maybe I'm a really good storyteller - I want my Pendant to let me know that. Or maybe it can leverage third party data too...things like weather, stock market, key dates, etc.

Basically, give me interesting insights/patterns/correlations in my life that I wouldn't think to recognize or connect.

https://feedback.limitless.ai/b/kvj1nlve/feature-ideas/show-me-hidden-patterns-in-my-life

**Comment by skywinder** [comment]
This is also huge thing that requires RAG on our memories. how about make a milestone with RAG-based tasks?

**Comment by skywinder** [comment]
https://feedback.omi.me/p/show-me-hidden-patterns-in-my-life " please vote:)
